tools/pipelines/templates/build-npm-package.yml

```
@@ -550,6 +555,7 @@ stages:
               workingDir: ${{ parameters.buildDirectory }}
               customCommand: 'run ci:build:docs'
 
+        - ${{ if or(ne(parameters.publishDocs, false), ne(parameters.taskBuildDocs, false)) }}:
```

```suggestion
        - ${{ if or(eq(parameters.publishDocs, true), eq(parameters.taskBuildDocs, true)) }}:
```

this is equivalent, no? I think A=true OR B=true is easier to understand than A != false OR B != false

---

I thought the intention was "any non-falsy value". Although it's a typed param so it can really only be true/false... I don't remember if ADO's coercion into boolean is tricky. But if we're happy being strict about the boolean-ness of the param, I agree with Tyler's proposal.